LoopKitUI/View Controllers/SettingsNavigationViewController.swift

```diff
@@ -32,8 +33,8 @@ open class PumpManagerSettingsNavigationViewController: SettingsNavigationViewCo
 
     open weak var pumpManagerOnboardingDelegate: PumpManagerOnboardingDelegate?
 
-    open func notifySetup(pumpManager: PumpManagerUI, withFinalSettings settings: PumpManagerSetupSettings) {
-        pumpManagerOnboardingDelegate?.pumpManagerOnboarding(didOnboardPumpManager: pumpManager, withFinalSettings: settings)
+    open func notifySetup(pumpManager: PumpManagerUI, basalSchedule: BasalRateSchedule) {
```

Comment:
It isn't clear to me why `basalSchedule` is here?

Reply:
It appears that `notifySetup` is not used at all. Thanks!